polynote-kernel/src/main/scala/polynote/kernel/dependency/CoursierFetcher.scala

```diff
@@ -167,30 +167,49 @@ object CoursierFetcher {
       .catchAll(recover)
   }.flatten  // this is pretty lazy, it's just so we can throw an exception in the main block.
 
+  private def whichClassifiers: RIO[Config, Set[coursier.Classifier]] = for {
+    config   <- Config.access
+  } yield if (config.downloadSources) Set(Classifier.sources) else Set.empty
+
+  private def artifacts(resolution: Resolution, cache: FileCache[ArtifactTask]) = for {
+    runtime          <- ZIO.runtime[Blocking]
+    blockingExecutor  = runtime.environment.get.blockingExecutor.asEC
+    classifiers      <- whichClassifiers
+  } yield Artifacts(new TaskManagedCache(cache, blockingExecutor))
+    .withResolution(resolution)
+    .withClassifiers(classifiers)
+    .withMainArtifacts(true)
+
+  private def isClasspath(pub: Publication) = pub.classifier.isEmpty || pub.classifier.value == "all"
```

Comment:
What does `isClasspath` mean here? What other values could `pub.classifier` have?

Reply:
Yeah... I agree it's kind of confusing.

Reply:
Worth a comment?